activitysim/core/configuration/logit.py

```diff
@@ -78,6 +78,17 @@ class BaseLogitComponentSettings(PydanticReadable):
     sharrow_skip: bool = False
     """Skip sharrow when evaluating this component."""
 
+    sharrow_fastmath: bool = True
```

Comment:
Do we want a top level setting that we can turn it off for all models?  If I understand the code right now, to turn it off we would have to add this to all model yamls, right?

Reply:
Yes, to turn fastmath off globally you'd need to turn it off in every component.  We can consider adding that in the future if it is desirable.